earthpy/tests/test_spatial.py

```diff
@@ -114,37 +114,44 @@ def test_extent_to_json():
 
 def test_bytescale_high_low_val():
     """"Unit tests for earthpy.spatial.bytescale """
-    
+
     arr = np.random.randint(300, size=(10,10))
-    
+
     # Bad high value
     with pytest.raises(ValueError, message="`high` should be less than or equal to 255."):
         es.bytescale(arr, high=300)
-        
+
     # Bad low value
     with pytest.raises(ValueError, message="`low` should be greater than or equal to 0."):
         es.bytescale(arr, low=-100)
-        
+
     # High value is less than low value
     with pytest.raises(ValueError, message="`high` should be greater than or equal to `low`."):
         es.bytescale(arr, high=100, low=150)
-        
+
     # Valid case. should also take care of if statements for cmin/cmax
     val_arr = es.bytescale(arr, high=255, low=0)
-    
+
     assert val_arr.min() == 0
     assert val_arr.max() == 255
-          
+
     # Test scale value max is less than min
     with pytest.raises(ValueError, message="`cmax` should be larger than `cmin`."):
         es.bytescale(arr, cmin=100, cmax=50)
-        
+
     # Test scale value max is less equal to min. Commented out for now because it breaks stuff somehow.
     with pytest.raises(ValueError, message="`cmax` and `cmin` should not be the same value. Please specify `cmax` > `cmin`"):
-        es.bytescale(arr, cmin=100, cmax=100)    
-        
+        es.bytescale(arr, cmin=100, cmax=100)
+
     # Test scale value max is less equal to min
     scale_arr = es.bytescale(arr, cmin=10, cmax=240)
-    
+
     assert scale_arr.min() == 0
     assert scale_arr.max() == 255
+
+
+def test_stack_invalid_out_paths_raise_errors():
```

Comment:
@mbjoseph do we also need to add a test for a user trying to write to the CWD? essentially what you are fixing here in this PR?

Reply:
There are a lot of tests we need to write for stack_raster_tifs - this test is just a start!

Reply:
ok wonderful. @joemcglinchy is working on these tests. my apologies i am forgetting the intent of this PR!

Reply:
excuse me if I have missed this. Does the update allow for writing to CWD??

Reply:
@joemcglinchy this PR allows users to write to the CWD by specifying an output path as `"output.tif"`, instead of `"./output.tif"`. The latter is the only way to do it currently, and both will be possible after merging this PR.

Reply:
💯